libraries/matrix/test/src/main/kotlin/io/element/android/libraries/matrix/test/TestData.kt

```diff
@@ -29,8 +29,8 @@ const val A_PASSWORD = "password"
 
 val A_USER_ID = UserId("@alice:server.org")
 val A_USER_ID_2 = UserId("@bob:server.org")
-val A_SESSION_ID = SessionId(A_USER_ID.value)
-val A_SESSION_ID_2 = SessionId(A_USER_ID_2.value)
+val A_SESSION_ID = A_USER_ID
+val A_SESSION_ID_2 = A_USER_ID_2
```

Comment:
Either we remove the type `SessionId`, and we only use `UserId`, or I would prefer the previous version of the code.